hack/make.sh

```diff
@@ -322,6 +322,17 @@ main() {
 		bundle "$bundle"
 		echo
 	done
+
+	# Add nested executables to bundle dir so we have complete set of
```

Comment:
I think I would have put that in the `binary` bundle (`hack/make/binary`) but either way is fine :wink: